### ISSUE TITLE
Keyerror:"XX"

### ISSUE THREAD

**Issue by mingxingpu** [princeton-vl/attach-juxtapose-parser#4]
I got the Keyerror:"XX" like this when I wanted to parse a sentence that only includs one word "grandpa".  What inconceivable is the word "grandma" can run correctly.   I don't know the reason. Can you help me ?
![image](https://user-images.githubusercontent.com/59463530/201046644-aad532cf-dfcf-4493-9e9e-5b0c5fe346bd.png)

**Comment by yangky11** [comment]
Could you include more detailed instructions on how to reproduce the issue? E.g., the commands to run, the input files.

**Comment by mingxingpu** [comment]
> Could you include more detailed instructions on how to reproduce the issue? E.g., the commands to run, the input files.
Firstly, I modified the input_examples.txt in the project like this: 
![image](https://user-images.githubusercontent.com/59463530/201244472-c4bd2f3e-dc9d-4170-b2f6-8c05b7b78e78.png)
And then, I modified the parse.yaml in the project like this:
![image](https://user-images.githubusercontent.com/59463530/201245328-0e64ad99-9faf-4130-b8e5-b5c6682778fa.png)
Finally, I ran a command: python parse.py.  Unfortunately, I got a KeyError.

What's interesting is that if I modifiy the input_examples.txt like this(the difference is just a dot):
![image](https://user-images.githubusercontent.com/59463530/201246066-b40e879c-fb1a-4ae6-818f-6ac7d4b70338.png)
It could work, and produce a result:
![image](https://user-images.githubusercontent.com/59463530/201246898-efbd38ab-08b2-4890-9049-6742c03df028.png)

**Comment by yangky11** [comment]
I will look into it and hopefully get back to you in one or two weeks.

**Comment by mingxingpu** [comment]
> I will look into it and hopefully get back to you in one or two weeks.

Many thanks for your kind and warm help.

**Comment by yangky11** [comment]
I tried both `grandpa` and `grandpa.` but was unable to reproduce this error.

For `grandpa`, I got `(NP (NN grandpa))`.
For `grandpa.`, I got `(NP (NN grandpa) (. .))`.

It's weird that the `.` is missing in your `grandpa.` example.

From your error message, it seems that an illegal `XX` POS tag was accidentally introduced. I suggest you looking into the POS tagging code to figure out where it came from.
https://github.com/princeton-vl/attach-juxtapose-parser/blob/eb85d61fb10ea3d9820de68e996fb5039c2c80a0/parse.py#L50-L52
